<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>432</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Quản lý nhân sự</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>791</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>401</width>
       <height>571</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>2</y>
        <width>391</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,stop: 1 #D0E5F9, stop:0.5 #B2B3F5 ,stop: 1 #9F6FF5,stop: 1
 #D587DB);
color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Quản lý nhân sự</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Nhóm 5</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>391</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="locale">
      <locale language="Vietnamese" country="Vietnam"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Đăng nhập</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Tài khoản:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Mật khẩu:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_login">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
border: 2px solid;
background: #7478CA;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Đăng nhập</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_user">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>201</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid;
color: black;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_password">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>201</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid;
color: black;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Hotline: 0362 764 100 (Mrs.Hảo)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
